bofire/data_models/surrogates/single_task_gp.py

```diff
@@ -28,3 +34,51 @@ class SingleTaskGPSurrogate(BotorchSurrogate):
     )
     noise_prior: AnyPrior = Field(default_factory=lambda: BOTORCH_NOISE_PRIOR())
     scaler: ScalerEnum = ScalerEnum.NORMALIZE
+
+    @root_validator
+    def validate_scaler_and_kernel(cls, values):
+        # Identify if TanimotoKernel is used at any point in the kernel
+        def dict_generator(dic, pre=None):
```

Comment:
Was designed to search through the kernel provided by the user to identify if TanimotoKernel was used. This has since been removed in the latest commit after the addition of TanimotoGPSurrogate, which defaults to the right kernel and scaler.